indico_zodbimport/modules/event_papers.py

```diff
@@ -169,6 +223,87 @@ def _migrate_paper_roles(self, contribution):
             elif role.role == PaperReviewingRoleType.editor:
                 contribution.paper_layout_reviewers.add(role.user)
 
+    def _migrate_review(self, contribution, old_judgment, review_type):
+        # Consider legacy custom states the same as "to be corrected"
+        proposed_action = JUDGMENT_STATE_PAPER_ACTION_MAP.get(int(old_judgment._judgement._id),
+                                                              PaperAction.to_be_corrected)
+        review = PaperReview(user=old_judgment._author.user, comment=old_judgment._comments, type=review_type,
+                             proposed_action=proposed_action, created_dt=_to_utc(old_judgment._submissionDate))
+        for old_answer in old_judgment._answers:
+            old_question = old_answer._question
+            question = self.legacy_question_map.get(old_question, None)
```

Comment:
`None` is already the default and it looks like a good case to actually catch the `KeyError` from `[]` instead